packages/valaxy-theme-yun/styles/layout/post.scss

```diff
@@ -5,6 +5,23 @@
   max-width: var(--yun-post-card-max-width);
 }
 
+.post-card-image {
+  @extend .post-card;
+  height: var(--yun-post-card-height);
+
+  @media screen and (max-width: 768px) {
```

Comment:
Use https://github.com/YunYouJun/valaxy/blob/main/packages/valaxy/src/client/styles/mixins/size.scss for media query.

Reply:
```scss
@include mobile {
  .post-card-info {
    flex-direction: column;
  }

   img {
    width: 100%;
  }
}
```
Like this?